src/components/UserPicker/UserPicker.tsx

```diff
@@ -1,26 +1,49 @@
 /*! Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT license. */
 import '../../yamui';
-import { IBasePickerProps, IBasePickerSuggestionsProps, NormalPeoplePicker } from 'office-ui-fabric-react/lib/Pickers';
+import {
+  IBasePickerProps,
+  IBasePickerSuggestionsProps,
+  BasePicker,
+  IInputProps,
+} from 'office-ui-fabric-react/lib/Pickers';
 import * as React from 'react';
 import { IPersona } from 'office-ui-fabric-react/lib/Persona';
+import { SuggestionItem, SuggestionsListItem } from '../SuggestionsList';
 
 export { IPersona, IBasePickerProps, IBasePickerSuggestionsProps };
 
+export interface UserPickerProps {
+  placeholder: IInputProps['placeholder'];
+  onResolveSuggestions: IBasePickerProps<SuggestionItem>['onResolveSuggestions'];
+}
+
 /**
  * The UserPicker is a thin wrapper of Fabric's
- * [NormalPeoplePicker](https://developer.microsoft.com/en-us/fabric#/components/peoplepicker) is used to select one
+ * [BasePicker](https://developer.microsoft.com/en-us/fabric#/components/peoplepicker) is used to select one
  * or more entities, such as people or groups.
  */
-export default class UserPicker extends React.Component<IBasePickerProps<IPersona>> {
+export default class UserPicker extends React.Component<UserPickerProps> {
   public render() {
-    const { onResolveSuggestions } = this.props;
     const suggestionsClass: IBasePickerSuggestionsProps = { className: 'y-userpicker-suggestions__container' };
     return (
-      <NormalPeoplePicker
+      <BasePicker
+        {...this.props}
+        onResolveSuggestions={this.onResolveSuggestions}
         className="y-userpicker__container"
         pickerSuggestionsProps={suggestionsClass}
-        onResolveSuggestions={onResolveSuggestions}
+        onRenderSuggestionsItem={this.onRenderSuggestions}
       />
     );
   }
+  private onRenderSuggestions = (props: any, itemProps: any) => {
+    return <SuggestionsListItem isSelected={true} onSelect={this.onSelect} />;
```

Comment:
@richardtagger Hmm, will I also need to handle bridging events like current selection, or will I need to make isSelected and onSelect optional props. Should I reconsider going away from the IPersona bound approach and just map props on SuggestionItem properties to the IPersona properties...

Reply:
How does BasePicker work with the JSX you pass - does it wrap it all in a click handler or?

Reply:
`isSelected` - In BasePicker how does keyboard navigation work? If FocusZone or something else is taking care of things we should make this optional

I'm fine with the IPersona approach, though the point of YamUI is to look Yammery. It might not matter by this point however